sdk/component/store/local/sqlite/store.go

```diff
@@ -113,21 +116,61 @@ func (m *manager) Validate(*ocsf.VulnerabilityFinding) error {
 
 // Read finds Vulnerability Findings by instanceID.
 // It returns ErrNoFindingsFound is not vulnerabilities were found.
-func (m *manager) Read(ctx context.Context, instanceID uuid.UUID) ([]*vf.VulnerabilityFinding, error) {
-	rawFindings, err := m.queries.FindingsByID(ctx, instanceID.String())
-	if err != nil {
-		return nil, errors.Errorf("could not find findings: %w", err)
+func (m *manager) Read(
```

Comment:
I'd consider always using the paginated method and just using some nice defaults to deal with it in case, so we can remove a lot of code and `FindingsByID`

Reply:
that would mean through a lot more extensive reworking of all the components that are calling it. having the query opts set to nil means that the caller doesn't care about how things are fetched but let's the storage layer do its own thing, meaning that it can choose to paginate or not and return the complete results